bom/contrib/jaxb/pom.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.openhab.core.contrib</groupId>
```

Comment:
obsolete if parent is used

Reply:
I did not use the parent to get this groupId. Do you think that `org.openhab.core.bom` would be the better groupId? Then I agree that the artifactId shpuld include `org.openhab.core.contrib`.